Documentation/artifact-format.md

```diff
@@ -281,29 +281,30 @@ Format: JSON
 
 ```
 {
-  "updates": [
-    {
-      "type": "rootfs-image"
-    },
-    {
-      "type": "rootfs-image"
+    "updates": [
+        {
+            "type": "rootfs-image"
+        },
+        {
+            "type": "delta-image"
+        }
+    ],
+    "artifact_provides": {
+        "artifact_name": "release-2",
+        "artifact_group": "fix",
+        "update_types_supported": [
+            "rootfs-image",
+            "delta-update"
+        ]
     },
-    {
-      ...
+    "artifact_depends": {
+        "artifact_name": "release-1",
+        "device_type": [
+            "vexpress-qemu",
+             "beaglebone"
+        ]
     }
-  ],
-  "artifact_provides": [
-          { "artifact_name": "name" },
-          { "artifact_group": "group-1" },
-          { "update_types_supported": ["rootfs-image"]},
-      ],
-  "artifact_depends": [
-          { "device_type": ["vexpress-qemu", "beaglebone"] },
-          { "artifact_name": ["rootfs-1"] },
-      ],
-  ],
-}
-```
+}```
```

Comment:
It's usual markdown style to have the triple backticks on their own line. Not sure if the rendering is affected, but just convention.

Reply:
Oh, and the rendering *is* affected, I see now!